### ISSUE TITLE
Display multiple data on the same graph

### ISSUE THREAD

**Issue by fvisticot** [freshking/DynamicGraphView#2]
Suppose I need to display the real time data from multiple sensors on same type every second.
Is it possible to display that on the same graph with the current framework ?

**Comment by freshking** [comment]
No, not at the moment. But its a great idea. The problem ist that you would
have to tag each value for the graphview to know which value to assignt to
which graphline.
But that shouldn't be hard to integrate into the current code.

Cheers,

Bastian Kohlbauer

Am 18.06.2013 um 23:11 schrieb fvisticot notifications@github.com:

Suppose I need to display the real time data from multiple sensors on same
type every second.
Is it possible to display that on the same graph with the current framework
?

—
Reply to this email directly or view it on
GitHubhttps://github.com/freshking/DynamicGraphView/issues/2
.

**Comment by freshking** [comment]
What you could do is to overlay multiple graphviews with clear backgrounds
to display multiple graphlines.

Am 18.06.2013 um 23:11 schrieb fvisticot notifications@github.com:

Suppose I need to display the real time data from multiple sensors on same
type every second.
Is it possible to display that on the same graph with the current framework
?

—
Reply to this email directly or view it on
GitHubhttps://github.com/freshking/DynamicGraphView/issues/2
.

**Comment by spromicky** [comment]
U can use this https://code.google.com/p/core-plot/ framework. Its very powerful.
